### ISSUE TITLE
ONestedRollbackException Issue

### ISSUE THREAD

**Issue by heleyboo** [orientechnologies/orientdb#6954]
We are facing the following issue when I run SQL to create Edge "CREATE EDGE inquiry FROM #29:5093 TO #25:17741 SET when = date()". Please help to solve it.

https://gist.github.com/heleyboo/6e94d7f87526ed9a81562fab71c862f3

## OrientDB Version, operating system, or hardware.
- v2.0 SNAPSHOT[ ] - .18[ ] .17[ ] .16[ ] .15[ ] .14[ ] .13[ ] .12[ ] .11[ ] .10[ ] .9[ ] .8[ ] .7[ ] .6[ ] .5[ ] .4[ ] .3[ ] .2[ ] .1[ ] .0[ ]
- v2.1 SNAPSHOT[ ] - .16[ ] .15[ ] .14[ ] .13[ ] .12[ ] .11[ ] .10[ ] .9[ ] .8[ ] .7[ ] .6[ ] .5[ ] .4[ ] .3[ ] .2[ ] .1[ ] .0[ ]
- v2.2 SNAPSHOT[x] - .rc1[ ] .beta2[ ] .beta1[ ]

## Operating System
- [x] Linux
- [ ] MacOSX
- [ ] Windows
- [ ] Other Unix
- [ ] Other, name?

**Comment by laa** [comment]
@heleyboo which version do you use ?

**Comment by heleyboo** [comment]
@laa I use OrientDB latest version(OrientDB 2.2.13 GA Community Edition)

**Comment by laa** [comment]
@heleyboo did you have any OOM or similar kind of exceptions before this problem happens ? 
Was database created under this version ?  Could you send database for investigation or at least files for cluster "inquiry_1" ?

**Comment by heleyboo** [comment]
Hi @laa,

> Did you have any OOM or similar kind of exceptions before this problem happens?

I did not encounter these kind of exceptions before this problem happens.

> Was database created under this version?

I created my database under version 2.2.12. When I encountered this problem. I copy the database an run it on v 2.2.13 and the problem still happen. But when I export data, and import data to new database, the problem doesn't happen again.

 I found some files relative to 'inquiry_1' in my database. Sorry because I can not give you all database.
[inquiry.zip](https://github.com/orientechnologies/orientdb/files/624113/inquiry.zip)
